src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The `href` attribute in the `link` tag should link to a CSS file, not an SCSS file. SCSS is not understood by browsers and needs to be compiled into CSS. Please check your build process and ensure that you are linking to the compiled CSS file.